wiki/_README.md

```diff
@@ -0,0 +1,53 @@
+_Certified fresh 230808 by Graham Johnson._
+
+# Navigating the docs
+
+For a complete list of entries, see our [Table of Contents](_TOC.md)
+
+# Updating the docs: How & when
+
+Whenever a code change PR breaks (or makes obsolete) existing documentation, the relevant entry should be updated. This docs update should be included within the PR itself, as part of the PR—just as one would update unit tests alongside a breaking code change.
+
+Similarly, whenever a PR adds codebase functionality where either:
+
+1. accompanying documentation has been explicitly requested 
+2. the implementing engineer believes the new functionality will not be easily legible to or extendable by future engineers
+
+...then that PR should include a new documentation entry as part of its purview.
+
+At least for the duration of 2023, as we implement this new system, any PR that updates or contributes to our documentation wiki should tag @gdjohnson (the documentarian) on Github.
+
+# Tracking documentation status
+
+An important part of maintaining a working documentation system involves flagging (1) areas in need of documentation (2) areas whose documentation has gone stale.
+
+For now, the best system is notifying the archivist, Graham Johnson, on Slack or Github (@gdjohnson). Down the line, a more structured system may be worth implementing.
+
+## "Certified fresh" datestamps
+
+Moving forward, our goal is to keep a “Certified fresh” datestamp on all wiki entries. This timestamp is accompanied by an optional note and the name of the certifier. For consistency, dates should take the format YYMMDD.
+
+Example certification:
+> _Certified fresh 230731 by Graham Johnson._
+> _Note: The optional super-admin steps listed under the "Verifying addresses" sub-section were not re-tested and may be stale._
+
+### What does “Certified fresh” mean operationally?
+
+1. "Certified fresh on X date" means that an engineer or documentarian has successfully tested or followed the entry’s documentation on X date.
+2. Successfully acting upon a few tidbits of information on an entry page for development guidance is not sufficient to certify an entry. 
+3. "Certified fresh" is a _strong recommendation_ from the certifying team member that the documentation can be faithfully followed.
```

Comment:
Who is responsible for maintaining fresh documentation about a particular module or part of the codebase?